src/javascript/binary/pages/trade/price.js

```diff
@@ -180,22 +179,6 @@ var Price = (function () {
             description.innerHTML = '<div>'+proposal['longcode']+'</div>';
         }
 
-        if (document.getElementById('websocket_form')) {
```

Comment:
@raunakkathuria JY asked that we don't hide the boxes anymore on invalid input for the new trading page.